test/gMAM.jl

```diff
@@ -0,0 +1,18 @@
+using CriticalTransitions, StaticArrays, Test
+
+function meier_stein(u, p, t) # in-place
+    x, y = u
+    dx = x-x^3 -10*x*y^2
+    dy = -(1+x^2)*y
+    SA[dx, dy]
+end
+σ = 0.25
+sys = StochSystem(meier_stein, [], zeros(2), σ, idfunc, nothing, I(2), "WhiteGauss")
+
+# initial path: parabola
+xx = range(-1.0,1.0, length=30)
+yy = 0.3 .* (- xx .^ 2 .+ 1)
+init = Matrix([xx yy]')
+
+gm = geometric_min_action_method(sys, init, maxiter=1000, showprogress=false)
+@test all(isapprox.(path[2,:][end-5:end], 0, atol=1e-3))
```

Comment:
Oops, sorry was too fast. It could be nice if we at a CI.yml who runs the tests. I agree that `isapprox(gm[2][end], 0.338, atol=1e-3)` is a better test. Will change the value to `0.3375`, as that is what it converges to when I do a long run.